.github/workflows/ruby-qltest.yml

```diff
@@ -29,6 +29,10 @@ defaults:
   run:
     working-directory: ruby
 
+permissions:
+  contents: read
+  security-events: read
```

Comment:
?

```suggestion
```

Reply:
Maybe?

I can't get ruby to build (I think I need larger runners or to try to kill a bunch of processes to get my runs more memory):
https://github.com/check-spelling-sandbox/codeql/actions/runs/7820458922/job/21335248948

We'll see what things look like when I update this PR.